modules/simple-payments/simple-payments.css

```diff
@@ -1,21 +1,57 @@
-body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message {
+
+.jetpack-simple-payments-wrapper {
+	margin: 0.5em 0;
+}
+
+.jetpack-simple-payments-title {
+	font-weight: bold;
+	display: block;
+	margin-bottom: 0.3em;
+}
+
+.jetpack-simple-payments-payment-row {
+	margin-top: 0.3em;
+}
+
+.jetpack-simple-payments-price {
```

Comment:
We could use flexbox (like we do in Calypso Editor previews) or at least clear the floats in the parent `div`.